nautobot/dcim/models/devices.py

```diff
@@ -1425,6 +1432,10 @@ def clean(self):
                     {"location": f'Devices may not associate to locations of type "{self.location.location_type}".'}
                 )
 
+    @property
+    def capabilities_list(self):
+        return array_to_string(self.capabilities)
```

Comment:
Feels weird to me to have a property called `_list` that returns a string :) Would `capabilities_string` be a better name?

Reply:
Yes it does, though this is a pattern that RackReservation.unit_list and Service.port_list have.